scripts/inspect-log-analysis/analyze_resample.py

```
@@ -0,0 +1,227 @@
+        original_tool=original_tool,
+        original_args=original_args,
+        name_match_rate=(name_matches / n) if n else 0.0,
+        exact_match_rate=(exact_matches / n) if n else 0.0,
+        no_tool_call_rate=(no_call / n) if n else 0.0,
+        errors=errors,
+    )
+
+
+def format_report(r: ResampleReport, text_chars: int = 500) -> str:
+    lines: list[str] = []
+    lines.append(f"# Resample report")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)